streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamsPartitionAssignor.java

```diff
@@ -768,15 +775,21 @@ private void assignTasksToClients(final Cluster fullMetadata,
 
         final Optional<org.apache.kafka.streams.processor.assignment.TaskAssignor> userTaskAssignor =
             userTaskAssignorSupplier.get();
+        UserTaskAssignmentListener userTaskAssignmentListener = (GroupAssignment assignment, GroupSubscription subscription) -> { };
```

Comment:
nit (address in PR 9): Streams coding style tip, prefer final variables whenever possible. So this would be like:
```
final UserTaskAssignmentListener listener;
if (userTaskAssignor.isPresent() {
    // do stuff
    listener = ...;
else {
   // do other stuff
   listener = (a, s) -> { };
}
```

Reply:
You can't change it once you make it final.